src/Plugin/views/argument_default/EntityRenderHistory.php

```diff
@@ -20,6 +21,23 @@ class EntityRenderHistory extends ArgumentDefaultPluginBase {
 
   use EntityRenderHistoryTrait;
 
+  /**
+   * The service container.
+   *
+   * @var \Symfony\Component\DependencyInjection\ContainerInterface
+   */
+  protected $container;
+
+  /**
+   * Constructs a container aware event dispatcher.
+   *
+   * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
+   *   The service container.
+   */
+  public function __construct(ContainerInterface $container) {
+    $this->container = $container;
```

Comment:
should inject entity type manager, not the container.

Reply:
@fago not sure 

maybe i don't fully get it 

so it's about replace `\Drupal::entityTypeManager();` with "injected"
Do you think it's ok for this case to have `entity type manager` maybe in a `create`  or something?

so not sure here

Reply:
instead of injecting the container, it must inject $this->entityTypeManager